docs/Modules.rst

```diff
@@ -660,6 +662,31 @@ Note that, if ``Derived.h`` includes ``Base.h``, one can simply use a wildcard e
   compatibility for programs that rely on transitive inclusion (i.e.,
   all of them).
 
+Re-export Declaration
+~~~~~~~~~~~~~~~~~~
+An *export-as-declaration* specifies that the current module is a private
+module whose interface will be re-exported by the named public module.
```

Comment:
Should we make this general instead of tied to "private modules"? Since private modules are currently a logical separation in the SDK that we don't enforce in the implementation (besides some magic with modulemaps), it might make more sense to make this general and only mention the private example as one of the use cases.

Reply:
Yeah, fixed for the version that went to upstream Clang. Thank you!